bugbot/rules/not_landed.py

```diff
@@ -232,6 +232,20 @@ def attachment_handler(attachments, data):
 
             if "phab" in res:
                 if res["phab"]:
+                    # Check for stackGraph dependencies
+                    rev = PHAB_URL_PAT.search(
+                        base64.b64decode(attachment["data"]).decode("utf-8")
+                    ).group(1)
+                    try:
+                        revision_data = self.phab.load_revision(rev_id=int(rev))
+                        stack_graph = revision_data.get("fields", {}).get(
+                            "stackGraph", {}
+                        )
+                        current_revision_phid = revision_data.get("phid")
+                        dependencies = stack_graph.get(current_revision_phid, [])
```

Comment:
De we need to use `.get` here?

Reply:
Did some testing, and it seems to work as intended without using `.get()`. Changed here: 133e960a45499c75452919669cfa668c3711c86d!